docs/docs/30-administration/15-agent-config.md

```diff
@@ -153,4 +153,29 @@ Configures if the gRPC server certificate should be verified, only valid when `W
 ### `WOODPECKER_BACKEND`
 > Default: `auto-detect`
 
-Configures the backend engine to run pipelines on. Possible values are `auto-detect`, `docker`, or `local`.
+Configures the backend engine to run pipelines on. Possible values are `auto-detect`, `docker`, `local` or `ssh`.
```

Comment:
Maybe we should merge #851 first to be able to create a doc entry for this new backend as well. 

CC @6543

Reply:
@qwerty287 @mscherer #851 got merged :)

would be nice if you could add some hints to this specific backend and describe where it should be used and where not

Reply:
https://github.com/woodpecker-ci/woodpecker/pull/861/commits/8f2941ff6480456fd520d905dd3132dab94502e6